server/cluster.go

```diff
@@ -503,6 +503,16 @@ func (c *Cluster) isRemoteTopic(topic string) bool {
 	return c.ring.Get(topic) != c.thisNodeName
 }
 
+// Returns remote node name where the topic is hosted.
+// If the topic is hosted locally, returns an empty string.
+func (c *Cluster) nodeNameForTopicIfRemote(topic string) string {
+	key := c.ring.Get(topic)
```

Comment:
I'm pretty sure it's going to crash when run standalone because `c` is `nil`.

Reply:
For sure. Fixed.